tuf/keys.py

```diff
@@ -832,6 +848,17 @@ def verify_signature(key_dict, signature, data):
       else:
         valid_signature = tuf.pycrypto_keys.verify_rsa_signature(sig, method,
                                                                  public, data) 
+    elif _RSA_CRYPTO_LIBRARY == 'pyca-cryptography': 
+      if 'pyca-cryptography' not in _available_crypto_libraries: # pragma: no cover
+        message = 'Metadata downloaded from the remote repository specified' +\
+          ' an RSA signature.  Verifying RSA signatures requires pyca/cryptography.' +\
+          '\n$ pip install cryptography, or pip install tuf[tools].'
+        raise tuf.UnsupportedLibraryError(message)
```

Comment:
I think the error message is wrong. *Your config* (from which `_RSA_CRYPTO_LIBRARY` was set) requires *this particular* lib. Generally speaking, either RSA lib would suffice.

Same goes for https://github.com/theupdateframework/tuf/pull/289/files?diff=split#diff-a3be85d1a9209356a591a86fedcad4a5R846 above.